cni/network/network.go

```diff
@@ -1073,7 +1076,15 @@ func (plugin *NetPlugin) Delete(args *cniSkel.CmdArgs) error {
 					zap.String("containerID", args.ContainerID))
 				sendEvent(plugin, fmt.Sprintf("Release ip by ContainerID (endpoint not found):%v", args.ContainerID))
 				if err = plugin.ipamInvoker.Delete(nil, nwCfg, args, nwInfo.Options); err != nil {
-					return plugin.RetriableError(fmt.Errorf("failed to release address(no endpoint): %w", err))
+					if errors.As(err, &connectionErr) {
```

Comment:
confused, why we need to add it here? I thought it was added in other PR already (in ipam.go)

Reply:
the changes in network.go are for azure-vnet. the other PR contains the same changes for azure-ipam

Reply:
this PR is CNI/CNS, other PR is `azure-ipam` since it is a separate Go module.

Reply:
This is a main function that will invoke different IPAM, for now, we should only handle it inside invoker_cns.go and that too only in AKS Swift scenario as only in that scenario CNS is cleaning up those ips latebound.

Reply:
The initial plan was to make the change for both azure-ipam and azure-vnet. The deadlock issue can still occur in the scenario with azure-ipam, at max pod saturation the pods in delete will get stuck if CNS is down. I think it would make sense to keep the changes as is, for each respective plugin